beacon-chain/blockchain/process_block.go

```diff
@@ -256,12 +256,10 @@ func (s *Service) onBlock(ctx context.Context, signed interfaces.SignedBeaconBlo
 			return
 		}
 
-		optimisticUpdate := lightclienthelpers.NewLightClientOptimisticUpdateFromUpdate(update)
-
 		// Return the result
-		result := &ethpbv2.LightClientOptimisticUpdateResponse{
+		result := &ethpbv2.LightClientUpdateResponse{
 			Version: ethpbv2.Version(signed.Version()),
-			Data:    optimisticUpdate,
+			Data:    update,
```

Comment:
Since we won't use NewLightClientOptimisticUpdateFromUpdate in previous line 259, and try to re-use the full version of update. 

We probably can achieve by manually set 
    update.nextCommittee = nil 
    .........
and only keep the fields should be kept for optimisitcUpdate